src/main/scala/com/microsoft/hyperspace/actions/VacuumAction.scala

```diff
@@ -49,4 +52,17 @@ class VacuumAction(
       }
     }
   }
+
+  final override protected def event(message: String): HyperspaceEvent = {
```

Comment:
why `protected`?

Reply:
This is implementation of the base class method `Action.event()`.  Are you saying to make this method public? Sorry did not understand he comment.

Reply:
I meant whether this can be `private`.

Reply:
Oh private is not possible. It becomes inaccessible to base `Action` class. I will then keep it protected. Thanks.